internal/bminventory/inventory.go

```diff
@@ -672,19 +632,21 @@ func (b *bareMetalInventory) InstallCluster(ctx context.Context, params installe
 	if err = b.db.Preload("Hosts", "status <> ?", host.HostStatusDisabled).First(&cluster, "id = ?", params.ClusterID).Error; err != nil {
 		return common.NewApiError(http.StatusNotFound, err)
 	}
-	if err = b.refreshAllHosts(ctx, &cluster); err != nil {
+	if _, err = b.clusterApi.RefreshStatus(ctx, &cluster, b.db); err != nil {
```

Comment:
Here there was refereshAllHosts.  It was replaced with RefreshCluster.  It should be in addition to refresh all hosts

Reply:
fixed